Brick-Studios/BeastArena
# Controller support

# Description

This PR adds controller support for players.
When using a controller the keyboard controls are disabled for that player.
Hotplug is supported, the controller will be assigned to the first player without one.

Use the controller branch for testing

Bindings:
Left control stick for moving left and right

| XBOX        | PS4           | NINTENDO  | action |
| ------------- | ------------- | ----- | ------------- |
| A  | Cross | B | Jump |
| X  | Square      |   Y | Shoot |
| B | Circle      |    A | Grab |


# How can this be tested?
Connect a controller and try to play! I have tested 3 which should be sufficient

# Checklist:
